.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
I had that in an earlier version of my PR for the map repo, too.

In fact this triggers as soon as somebody creates a new PR against the master branch.
I think that's too early ;-)

From what I read, when a PR is merged, it results in the above trigger "on push master" getting fired.
So these lines can be removed IMHO.

Reply:
Good point. I'll remove .travis.yml as it's not needed anymore.

No wonder the build_and_deploy.yml is similar to yours as honestly I've just copied and slightly adopted your file of the fprsmap repo PR ;-) So for sure I trust you and will remove these lines too.